content/departments/product-engineering/product/process/unofficial_product_sales_metrics_definitions.md

```diff
@@ -0,0 +1,45 @@
+# Unofficial product sales metrics definitions
+
+While there are multiple company departments working to standardize and unify per-product revenue and sales reporting, that shouldn't stop us from evaluating the sales patterns of add-on products like Code Insights and Batch Changes. These metrics help us track success, prioritize investments, determine when parts of our product and sales strategy need iteration, and learn from what's working well.
+
+Since these numbers are somewhat unofficial due to our current sales process, the most important thing is that we use the same definitions so we can summarize and learn across products.
+
+(Of course, all the numeric values on this page are wildly fake and do not reflect neither proportionally nor absolutely the price or metrics of Sourcegraph. They exist for example purposes only.)
+
+## Attach rate
+
+The percentage of new logos that include an add-on product. When calculating attach rate on a quarterly basis, we do count expansion sales TODO THINK ABOUT THIS. When calculating overall, it's just the percentage of total customers with an add-on product.
+
+**Example:** If we landed at 100 new customers and 30 of them purchased Batch Changes, that would be a 30% attach rate.
+
+**How we use this metric:** Attach rate provides signal on the breadth of product-market fit and the overall enablement readiness and sales motion of a product.
+
+## Per-product iARR
+
+The dollar value we attribute to the product sale.
+Until we have better tracking in place on the sales and finance side, we use the following heuristics, in order:
```

Comment:
```suggestion
Until we have better tracking in place on the sales and finance side, teams use roughly equivalent ARR metrics (tradeoff of time and information precision). Batch changes uses the % of ARR its sticker price suggests for all Batch Changes deals. Code Insights (since it is newer to market) uses the following heuristics, in order:
```

Reply:
@malomarrec what do you think about this?